qiskit/test/mock/utils/backend.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do you think there might be a better name for this module? Maybe fake_backend_builder?